pkgs/development/python-modules/pyfaidx/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "pyfaidx";
+  version = "0.5.5.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1akc8hk8rlw7sv07bv1n2r471acvmxwc57gb69frjpcwggf2phls";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/pyfaidx/";
+    description = "Python classes for indexing, retrieval, and in-place modification of FASTA files using a samtools compatible index";
+    license = licenses.bsdOriginal;
```

```suggestion
    license = licenses.bsd3;
```
https://github.com/mdshw5/pyfaidx/blob/master/LICENSE